shell/platform/darwin/ios/ios_context.mm

```diff
@@ -6,12 +6,9 @@
 #include "flutter/shell/platform/darwin/ios/rendering_api_selection.h"
 
 #include "flutter/fml/logging.h"
-#include "flutter/shell/platform/darwin/ios/ios_context_software.h"
-
-#if SHELL_ENABLE_METAL
 #include "flutter/shell/platform/darwin/ios/ios_context_metal_impeller.h"
 #include "flutter/shell/platform/darwin/ios/ios_context_metal_skia.h"
-#endif  // SHELL_ENABLE_METAL
+#include "flutter/shell/platform/darwin/ios/ios_context_software.h"
```

Comment:
Curious what this is still used for.

Reply:
`IOSContextSoftware`
https://github.com/flutter/engine/blob/03cd54d4285a44e1e082ba0d1277fbee5b3f4f48/shell/platform/darwin/ios/ios_context.mm#L34C7-L34C53